versioning/BRM/dbrm.cpp

```diff
@@ -4453,6 +4454,97 @@ void DBRM::deleteAISequence(uint32_t OID)
   }
 }
 
+void DBRM::addToLBIDList(uint32_t sessionID, vector<LBID_t>& lbidList)
```

Comment:
To be honest I didn't get the semantics of addToLBIDList(). Why do you expect lbidList to be different after rollBackTransaction() call in CommandPackageProcessor::processPackage(). 
My concern is that with a relatively huge EM addToLBIDList will become another perf killer for DML  IMHO b/c of multiple lookupLocal() and getExtents()(both have O(N) complexity where N is the EM size). Let's discuss why do we need this method im DBRM and how we can make it better.

Reply:
Refer to my reply in dbcon/dmlpackageproc/commandpackageprocessor.cpp where I explain why we need `addToLBIDList()`. Please note that this function is only invoked in case of a rollback so I am a bit less concerned about the performance implications, but sure let's discuss if there are ways to make it more efficient.